### PR TITLE
fix: cosmetic issues with tokens page with clockface beta, context menu and the tests

### LINKED ISSUE #2401: Tokens page style issues
1. Card context menu needs to change
2. Primary button width
3. The all buckets, scoped toggle is broken.
4. Bucket select all deselect all needs more space so overlay needs to be wider.

### PR DESCRIPTION
**Pull request by daviddkkim**
Closes #2401 

Tokens page context menu was changed. deprecating the context menu component in favor of clockface buttons.
Tests were fixed with the new context menu.
Overlay spacing changed.

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2619)

File: src/authorizations/components/TokenRow.tsx [hunk 1]
```diff
@@ -78,15 +78,27 @@ class TokenRow extends PureComponent<Props> {
...
+      /*  <Context>
         <Context.Menu icon={IconFont.Trash} color={ComponentColor.Danger}>
           <Context.Item
             label="Delete"
             action={this.handleDelete}
             testID="delete-token"
           />
         </Context.Menu>
-      </Context>
+      </Context> */
```
Comment (hoorayimhelping) [inline src/authorizations/components/TokenRow.tsx:89]: ✂️

**Review by hoorayimhelping — APPROVED** [review]
looks good, once you remove that comment block

**Review by blegesse-git — APPROVED** [review]

**Review by Palakp41 — APPROVED** [review]
LGTM

**Pull request merged**